Composer/packages/lib/code-editor/src/LuEditor.tsx

```diff
@@ -248,11 +250,12 @@ const LuEditor: React.FC<LULSPEditorProps> = (props) => {
   );
 
   const insertEntity = useCallback(
-    (entityName: string) => {
+    (entityName: string, entityType: string) => {
       if (editor) {
         const edits = computeInsertLuEntityEdits(entityName, editor);
         if (edits) {
           editor.executeEdits('toolbarMenu', edits);
+          telemetryClient.track('LUEditorEntityTagAdded', { entityType });
```

Comment:
We should differentiate between insert vs. tag
We should differentiate between tag via toolbar vs floating labeling menu

Reply:
entityType here is either: 'ml' or 'prebuilt', if you want the real prebuilt entity such as ```age```, you need to check if it's prebuilt, use entity name